gems/aws-sdk-core/spec/aws/cbor/decoder_spec.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require_relative '../../spec_helper'
+require 'aws-sdk-core/cbor/decoder'
```

Comment:
Why are all these required for specs to work?

Reply:
See other answer about engines. (we do an explicit require in the cbor engine when setting a default. this does not work with autoload).